docs/release-notes/3.0.0_Beta.md

```diff
@@ -0,0 +1,244 @@
+
+This simplified workflow combines with PMM's new ability to run database exporters on any PMM Client node rather than solely on PMM Server. This distributed approach enables monitoring of a larger number of remote and cloud-hosted databases like RDS instances by spreading the monitoring load across multiple PMM Clients.
+
+When adding new services through the PMM UI, you can now specify which PMM Client node will run the monitoring exporter. While PMM Server remains the default option, you can choose any node with PMM Client installed, including selecting specific clients on nodes with multiple installations.
+
+This distributed monitoring capability is currently available for new service additions, with support for existing service modification planned for a future release.
+
+For more details, see [Connect services](../install-pmm/install-pmm-client/index.md##connect-services).
+
+![Choose node](../_images/choose_node.png)
+
+### Simplified AWS installation process
+
+We've simplified AWS installations to match our standard Docker/Postman workflow. After installation, you'll immediately see the PMM login screen.
+
+Use **admin** as the username and your EC2 Instance ID as the password (the default PMM password cannot be used for security reasons).
+
+You can find your Instance ID in the AWS Console. For detailed instructions, see [Install PMM Server on AWS Marketplace](../install-pmm/install-pmm-server/aws/aws.md).
+
+### Official ARM support for PMM Client
+
+PMM 3 Beta now officially supports ARM architecture, upgrading from its experimental status in PMM 2.43. This means you can reliably monitor databases on ARM platforms, taking advantage of their cost-effective infrastructure and energy efficiency in data centers and cloud environments.
+
+Installation follows the standard PMM Client process, with no special requirements for ARM systems.  
+
+Test this feature and share your experience on the [PMM forum](https://forums.percona.com/c/percona-monitoring-and-management-pmm/percona-monitoring-and-management-pmm-v2/31) to help accelerate its path to GA.
+
+### Improved UX with Grafana's latest release
+
+PMM now integrates Grafana 10.4, which delivers the following important enhancements alongside all the advancements introduced since the previous Grafana 9.2.20 integration.
+
+For the full list of changes included in this update, see [Grafana's 10.4 release blog](https://grafana.com/blog/2024/03/06/grafana-10.4-release-all-the-latest-features/).
+
+## Monitoring improvements
+
+### Added monitoring support for default PostgreSQL database
+
+PMM now provides full monitoring support for its internal PostgreSQL databases, with metrics displayed across Query Analytics (QAN).
```

Comment:
it sounds like we are talking about internal PostgreSQL database in PMM, but it's about default PostgreSQL database on user instances called `postgres`.